cmd/codegen/arguments_wrapper.go

```diff
@@ -58,6 +58,7 @@ func argWrapper(argType string) (wrapper argumentWrapper, err error) {
 		"unsigned int*":            simplePtrW("uint32", "C.uint"),
 		"double":                   simpleW("float64", "C.double"),
 		"double*":                  simplePtrW("float64", "C.double"),
+		"const double*":            doubleArrayW,
```

Comment:
you can say `simplePtrSliceW("double", "float64")` instead of adding `doubleArrayW` function

Reply:
Tested `simplePtrSliceW("double", "float64")`

This:
``` go
func PlotSetupAxisTicksdoublePtr(axis PlotAxisEnum, values []float64, n_ticks int32) {
	C.wrap_ImPlot_SetupAxisTicks_doublePtr(C.ImAxis(axis), (*C.double)(&(values[0])), C.int(n_ticks))
}
```

Became this:
``` go
func PlotSetupAxisTicksdoublePtr(axis PlotAxisEnum, values *[]float64, n_ticks int32) {
	valuesArg := make([]double, len(*values))
	for i, valuesV := range *values {
		valuesArg[i] = double(valuesV)
	}

	C.wrap_ImPlot_SetupAxisTicks_doublePtr(C.ImAxis(axis), (*double)(&valuesArg[0]), C.int(n_ticks))

	func() {
		for i, valuesV := range valuesArg {
			(*values)[i] = float64(valuesV)
		}
	}()

}
```

Is it ok to  allocate new array on every call?

Reply:
It is a current scenerio for all the methods.
If it isn't OK, we should edit `simplePtrSlice` to change the code everywhere, not only for []float64

Reply:
It's ok. Looks safer to use.
Later I will suggest some optimizations there.

Reply:
Looks it's not ok :)
```
# github.com/AllenDang/cimgui-go
../cimgui-go/cimplot_funcs.go:1682:22: undefined: double
../cimgui-go/cimplot_funcs.go:1684:18: undefined: double
../cimgui-go/cimplot_funcs.go:1696:49: undefined: double
../cimgui-go/cimplot_funcs.go:1823:22: undefined: double
../cimgui-go/cimplot_funcs.go:1825:18: undefined: double
../cimgui-go/cimplot_funcs.go:1837:45: undefined: double
../cimgui-go/cimplot_funcs.go:1959:22: undefined: double
../cimgui-go/cimplot_funcs.go:1961:18: undefined: double
../cimgui-go/cimplot_funcs.go:1973:49: undefined: double
../cimgui-go/cimplot_funcs.go:1987:22: undefined: double
../cimgui-go/cimplot_funcs.go:1987:22: too many errors
```

``` go
func PlotPlotInfLinesdoublePtr(label_id string, values *[]float64, count int32) {
	label_idArg, label_idFin := wrapString(label_id)
	valuesArg := make([]double, len(*values))
	for i, valuesV := range *values {
		valuesArg[i] = double(valuesV)
	}

	C.wrap_ImPlot_PlotInfLines_doublePtr(label_idArg, (*double)(&valuesArg[0]), C.int(count))

	label_idFin()
	func() {
		for i, valuesV := range valuesArg {
			(*values)[i] = float64(valuesV)
		}
	}()

}
```

does `double(valuesV)` should be `C.Double(valuesV)`?

Reply:
> It's ok. Looks safer to use.

What do you mean by "safer to use"? We should follow standards in the rest of code in `arguments_wrapper.go`.
In #82 argument wrappers has been generalized so that we don't need separated function for each go type anymore.
Thats why you should use `simplePtrSliceW` here and (if you are able to optimalize that method) do it once for all the types.
Anyway, pleas change this line to
```go
		"const double*":            simplePtrSliceW("double", "float64"),
```

Reply:
yah. it should be `C.double`

Reply:
Replaced by `simplePtrSliceW("C.double", "float64")`